docs/library/forms/switch.md

```diff
@@ -0,0 +1,216 @@
+---
+components:
+    - rx.radix.themes.Switch
+---
+
+```python exec
+import reflex as rx
+from reflex.components.radix.themes.components import *
+from reflex.components.radix.themes.layout import *
+from reflex.components.radix.themes.typography import *
+from pcweb.templates.docpage import style_grid
+```
+
+# Switch
+
+
+A toggle switch alternative to the checkbox.
+
+## Basic Example
+
+```python demo
+text(
+    flex(
+        switch(default_checked=True),
```

Comment:
Maybe don't even set this prop for the first example

Reply:
I feel this is a good place to put it as it is so simple and it is probably going to be the most used prop